### ISSUE TITLE
JSON appears in Error List's Code filter menu popup

### ISSUE THREAD

**Issue by Jehoel** [dotnet/roslyn-analyzers#5047]
### Analyzer

Unknown

### Analyzer source

Unsure.

### Describe the bug

After using Visual Studio for more than an hour or so, hidden code analysis errors start to amass in the Error List's Code column filter popup, they appear as serialized JSON with zero occurrence counts. 

![image](https://user-images.githubusercontent.com/1693078/115474692-d608f180-a1f2-11eb-932d-d279805c841d.png)

An example entry is:

```
{"NewText": "Action ?", "EditRange": {"Item1":49,"Item2":32, "EndLocation":{"Item1":49,"Item2":38}},"ProgramIdentifier":"-1849195906_-1851706466", "EditType":0}
```

### Steps To Reproduce

1. Create a .NET Class Library project that multi-targets .NET Framework, .NET Core 3.1, and .NET Standard 2.0 and enable nullable reference types and C# 8.0 or later.
2. Add C# code that contains triggers static analysis warnings.
3. Spend an hour or so working on the C# code to address the prior warnings.
4. Open the Error List and open the Code column filter and you should at least see a few of these entries.

### Expected behavior

Them to not appear.

### Actual behavior

They appear.

**Comment by Youssef1313** [comment]
@mavasani This doesn't feel related to roslyn-analyzers. Should this move to roslyn?
